pkg/api/secretbootstrap/secretboostrap.go

```diff
@@ -18,23 +18,20 @@ const (
 	AttributeTypePassword AttributeType = "password"
 )
 
-type BitWardenContext struct {
-	BWItem               string                 `json:"bw_item"`
+type ItemContext struct {
+	Item                 string                 `json:"item"`
```

Comment:
Can we maybe just do a first pr that adds new fields with new names, then change the config, then remove the old fields?

Reply:
I was hoping that I can do a follow-up, after the changes in this PR merge, in the release repo changing the config. It won't cost anything. Worst case scenario we will have a failed periodic job.

Reply:
I can disable the periodic jobs, land the changes in this PR and in release repo, then enable the jobs again. WDYT?

Reply:
Not a fan of that, we also have presubmits that use this config and this is a gigantic PR that has a lot of potential to contain bugs, so I do not want to require it to be merged/reverted in lockstep with a config change. Please just leave the `json` annotation as it was, we can change it together with the config in a follow-up after this had some time to sink and we are confident it works correctly.

Also before this can merge, all usage of attachments and passwords in our config needs to be removed.

Reply:
Yea you are right. It's safer to do it like this. I will update the PR.